app/Resources/assets/css/application.scss

```diff
@@ -246,7 +246,7 @@ $tooltip-shadow: #ccc;
     }
 
     .bold {
-        font-weight: 500;
+        font-weight: bold;
```

Comment:
Is `bolder` better here? It is relative to the context.

Reply:
Yay, Sam is back on XTools! :)

I don't think I even knew about `bolder`. I don't even know where `500` came from... I just noticed after moving to Linux, it has no effect in Chromium. `bolder` does work, but it's not any bolder than `bold`. Anyway, here I think all we want is normal boldness. That was the intention, anyway.